models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -53,6 +54,7 @@ kyberswap_dex AS (
         ,if(starts_with(cast(t.deltaQty0 as varchar), '-'), p.token0, p.token1)                                  AS token_sold_address
         ,t.contract_address                                                                                      AS project_contract_address
         ,t.evt_tx_hash                                                                                           AS tx_hash
+        ,'elastic'                                                                                               AS version
```

Comment:
And for `elastic` version here?

Reply:
No we can skip it, I just update the schema to pass the test.